dm/loader/lightning.go

```diff
@@ -471,7 +472,12 @@ func (l *LightningLoader) Update(ctx context.Context, cfg *config.SubTaskConfig)
 func (l *LightningLoader) status() *pb.LoadStatus {
 	finished, total := l.core.Status()
 	progress := percent(finished, total, l.finish.Load())
-	currentSpeed := int64(l.speedRecorder.GetSpeed(float64(finished)))
+	finishedForSpeed := finished
+	// for tidb backend, `finished` is too big granularity, use below metric is much better
+	if l.cfg.LoaderConfig.ImportMode == config.LoadModeLogical {
+		finishedForSpeed = int64(metric.ReadCounter(l.core.Metrics().BytesCounter.WithLabelValues(metric.BytesStateRestored)))
```

Comment:
you can leave a comment about why BytesStateRestored can't be used for physical mode?